internal/node/config/generate/Config.toml

```
@@ -206,11 +206,11 @@ It is also possible to set the endpoint without a password and load it from Post
 See [this](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-PASSFILE)
 for more information."""
 
-[postgres.CARTESI_POSTGRES_SSLMODE_ENABLED]
-default = "true"
+[postgres.CARTESI_POSTGRES_SSL_MODE_DISABLE]
+default = "false"
```

Let's simplify it for now, so we can move on
```suggestion
[postgres.CARTESI_SSL_ENABLED]
default = "true"
```

We can revisit it later on